README.md

```diff
@@ -1,9 +1,47 @@
-# oxa-tools
 
-Deploying and maintaining Open edX on Azure
+# Open edX on Azure Deployment Guide
+
+This guide is for Open edX on Azure Deployment for the Learning as a Service (LaaS) program. There
+are three basic steps to on board into the LaaS program:
+1. Acceptance into the LaaS program
+2. Deploying your Open edX on Azure and
+3. Getting the Microsoft Certificates ready for users.
+This Deployment Guide covers getting your Open edX on Azure instance running (Step 2)
+
+![laas_program](images/figure-1.png "LaaS Program")
+
+*Figure 1: LaaS Program*
+
+# Deploying your Open edX on Azure
+This repo contains guides and tools designed to help you deploy and manage a highly available and scalable Open edX on Azure.
+If you have Azure account you can deploy Open edX via the Azure portal using the guidelines below. Please note that while you can use an Azure free account to get started depending on which configuration you choose you will likely be required to upgrade to a paid account.
+
+
+## Fully configurable deployment
+The number of configuration options might be overwhelming, so some pre-defined/restricted deployment options for typical Open edX scenarios follow this.
+
+## Predefined deployment options
+Below are a list of pre-defined/restricted deployment options based on typical deployment scenarios (i.e. dev/test, production etc.)
+
+| Deployment Type            | Description                                                                                                    | Environment Preferred |
+|----------------------------|----------------------------------------------------------------------------------------------------------------|-----------------------|
+| Minimal                    | Single machine instance                                                                                        | Development and Test  |
+| High availability instance | A production stack comprising of various Azure components as mentioned in LaaS Architecture | Production            |
 
 ## Deploying single machine instance (for development and test)
 
+### Server Requirements 
+The following server requirements will be fine for supporting hundreds of registered students on a single server.
+
+Note: This will run MySQL, Memcache, Mongo, nginx, and all of the Open edX services (LMS, Studio, Forums, ORA, etc) on a single server. In production configurations we recommend that these services run on different servers and that a load balancer be used for redundancy. Setting up production configurations is beyond the scope of this wiki page.
+
+* Ubuntu 16.04 amd64 (oraclejdk required). It may seem like other versions of Ubuntu will be fine, but they are not.  Only 16.04 is known to work.
+* Minimum 8GB of memory
+* At least one 2.00GHz CPU
+* Minimum 25GB of free disk, 50GB recommended for production level use
+
+### Installation Instructions
+
```

Comment:
Question: Since open-release/ficus.4 and open-release/ginkgo.2 are the latest releases of each, should we only support those versions?  Seems odd that we would ask partners to install older releases which may not contain various bug fixes.  Note however that this would require testing and perhaps some additional code changes on our part.